packages/atclient/src/request_options.c

```diff
@@ -1426,3 +1433,319 @@ void atclient_get_atkeys_request_options_unset_show_hidden(atclient_get_atkeys_r
   options->show_hidden = false;
   atclient_get_atkeys_request_options_set_show_hidden_initialized(options, false);
 }
+
+void atclient_pkam_authenticate_options_init(atclient_pkam_authenticate_options *options) {
+  memset(options, 0, sizeof(atclient_pkam_authenticate_options));
+  options->root_server_host = ATCLIENT_PKAM_AUTHENTICATE_DEFAULT_ROOT_SERVER_HOST;
+  options->root_server_port = ATCLIENT_PKAM_AUTHENTICATE_DEFAULT_ROOT_SERVER_PORT;
```

Comment:
Also, we don't call it the "root" server anymore. We call it the "atDirectory", and ideally, our new SDK is aligned with our new marketing terms